bin/metrics-http-json.rb

```diff
@@ -60,6 +60,11 @@ class HttpJsonGraphite < Sensu::Plugin::Metric::CLI::Graphite
          short: '-m METRIC::JSONKEY',
          long: '--metric METRIC::JSONKEY'
 
+  option :header,
```

Comment:
```suggestion
  option :headers,
```
As this indicates it supports multiple headers